bip-0009.mediawiki

```diff
@@ -119,10 +123,7 @@ other one simultaneously transitions to STARTED, which would mean both would dem
 
```

Comment:
This section is wrong. Also, the selection criteria above needs to be adjusted. See https://github.com/bitcoin/bitcoin/pull/21377#discussion_r611492633